api/renterhost_test.go

```diff
@@ -413,7 +469,7 @@ func TestRemoteFileRepair(t *testing.T) {
 // TestHostAndRentVanilla sets up an integration test where a host and renter
 // do basic uploads and downloads.
 func TestHostAndRentVanilla(t *testing.T) {
-	if testing.Short() || !build.VLONG {
+	if testing.Short() {
```

Comment:
and we should also definitely always be running at least one set of host+rent tests